.github/actions/javascript/getDeployPullRequestList/getDeployPullRequestList.js

```diff
@@ -18,13 +18,13 @@ const itemToFetch = isProductionDeploy ? 'release' : 'tag';
  */
 function getTagsOrReleases(fetchReleases) {
     if (fetchReleases) {
-        return octokit.repos.listReleases({
+        return GithubUtils.octokit.repos.listReleases({
```

Comment:
I think an alternative would be to do `octokit.rest.repos.listReleases()` ([ref](https://octokit.github.io/rest.js/v18#repos-list-releases)), but this was the only place we don't use GithubUtils, so I went with that for consistency.